site/app/libraries/database/DatabaseQueries.php

```diff
@@ -648,6 +648,48 @@ public function getTotalSubmittedUserCountByGradingSections($g_id, $sections, $s
         return $return;
     }
 
+    public function getTotalSubmittedTeamCountByGradingSections($g_id, $sections, $section_key) {
+        $return = array();
+        $params = array($g_id);
+        $where = "";
+        if (count($sections) > 0) {
+            // Expand out where clause
+            $sections_keys = array_values($sections);
+            $where = "WHERE {$section_key} IN (";
+            foreach($sections_keys as $section) {
+                $where .= "?" . ($section != $sections_keys[count($sections_keys)-1] ? "," : "");
+                array_push($params, $section);
+            }
```

Comment:
Seems like generating the placeholders is done everywhere else like:
`implode(",", array_fill(0, count($sections), "?"))`
and using `array_merge` on the `params` and `section_keys` will preserve order

Reply:
Yep, was just copying some old code with this.